torchprime/experimental/torchax_models/deepseek_v3/tests/test_prefill.py

```diff
@@ -1,23 +1,35 @@
-import pytest
+import unittest
 
+import torch
+import torchax
+import torchax.interop
+from torchprime.experimental.torchax_models.deepseek import model as ds_model
 
-# TODO(https://github.com/AI-Hypercomputer/torchprime/issues/75): Fix the failure on torch 2.6,
-# then enable the test unconditionally.
-@pytest.mark.deepseek
-def test_single_device_compile():
-  from torchprime.experimental.torchax_models.deepseek_v3.prefill_benchmark import (
-    single_device_compile,
-  )
 
-  single_device_compile()
+class DeepseekModuleTest(unittest.TestCase):
 
+  def setUp(self):
+    torchax.enable_globally()
 
-# TODO(https://github.com/AI-Hypercomputer/torchprime/issues/75): Fix the failure on torch 2.6,
-# then enable the test unconditionally.
-@pytest.mark.deepseek
-def test_single_device_eager():
-  from torchprime.experimental.torchax_models.deepseek_v3.prefill_benchmark import (
-    single_device_eager,
-  )
+  def tearDown(self):
+    torchax.disable_globally()
 
-  single_device_eager()
+  def test_moe_can_jit(self):
```

Comment:
Do we need the `os.environ["XLA_FLAGS"] = "--xla_force_host_platform_device_count=4"` thing since this test is running on the CPU?

Reply:
no need. as long as input tensors are not sharded the model code runs as if its single dev